polls/tests/test_voting.py

```diff
@@ -75,7 +75,7 @@ def test_voting_once_only(self):
         self.assertContains(response, 'is this a test')
         response = client.post(reverse('molo.polls:vote',
                                kwargs={'question_id': question.id}))
-        self.assertContains(response, "You didn&#39;t select a choice")
+        self.assertContains(response, "field is required")
```

Comment:
@codiebeulaine both. Keep the "you didn't select" error message and use `form.choice.errors` to show the error message